README.md

```diff
@@ -80,6 +98,13 @@ npm test
 - LinkedIn: [@HenryOdongo](https://www.linkedin.com/in/henry-odongo-91b830182/)
 - Website: [@Portfolio](https://odongo006.github.io/My-Portfolio-Project/)
 
+👤 **Mert**
+
+- GitHub: [@mertkantaroglu](https://github.com/mertkantaroglu)
+- Twitter: [@]()
+- LinkedIn: [@]()
+- Website: [@]()
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly you may try to add valid links to these social medias or You remove the unused ones to make your Readme more professional 👍 👍